contributors/devel/cherry-picks.md

```diff
@@ -25,18 +25,20 @@ task is for backporting PRs from master to release branches.
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestone approval needs to be added for the target release branch.
+   * The Patch Release Manager for the target release branch is responsible for
+   applying the `cherrypick-approved` label.
 
 ## Cherry Pick Review
 
-Cherry pick pull requests are reviewed differently than normal pull requests. In
-particular, they may be self-merged by the release branch owner without fanfare,
-in the case the release branch owner knows the cherry pick was already
-requested - this should not be the norm, but it may happen.
+Cherry pick pull requests have an additional requirement compared to normal pull
+requests. They must be approved specifically for cherry-pick by SIG leads.
```

Comment:
SIG leads' approval is not really required, unless during code slush where the SIG lead will need to apply the `status/approved-for-milestone` label.

But in general, anyone who has the power to `/approve` the relevant changes in the PR can approve the cherry-pick.

Reply:
> it's not that it takes a SIG lead; approvers have the power of adding the cherry-pick approval. Yes?

Sorry! I should have been clearer.

> Cherry pick pull requests have an additional requirement compared to normal pull
requests. They must be approved specifically for cherry-pick by SIG leads.

I meant that cherry-picks don't need any approval from SIG leads.

- To get the `approved` label, anyone within the approvers list of the relevant OWNERS file can `/approve`.
- To get the `cherrypick-approved` label, only the patch release manager can add the label.